### DISCUSSION TITLE
Inflection configuration [Dec 09]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4665] [Discord Threads]
> 🙋‍♂️ **lublak**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### <@100849859006840832> Inflection configuration can be done via UI during project creation.  
![image0](https://cdn.discordapp.com/attachments/1039474985460322335/1039744291016294420/Screenshot_2022-11-09_at_9.01.04_AM.png)  
#  
> 👨‍🦲 **dstala**  
#### There is no automatic way to add "Created" and "Updated" fields to existing table. It needs to be configured manually  
#  
> 🙋‍♂️ **lublak**  
#### yes I know that. It is about the setting of the installation. I have seen that some set it with json (currently also for you only possibility because with the url ssl problems arise, with the json I can set ssl: false).   
#### I was just wondering what the other things are in these json files that some people show.  
#  
> 🙋‍♂️ **lublak**  
#### <@857998597474484295> https://github.com/nocodb/nocodb/discussions/838#discussioncomment-1756309  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1039474985460322335)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
